config/checkstyle-checks.xml

```diff
@@ -617,6 +617,13 @@
       -->
       <property name="severity" value="ignore"/>
     </module>
+    <module name="UnusedTryResourceShouldBeUnnamed">
+      <!--
+        Unnamed try parameters will not compile on java versions before 21. As
+        we migrate to jdk 21 we will use this check.
```

Comment:
We can create issue on it, and simply put `until #xxxxxx ` in comment.
Migration to jdk21 will happen on January 2026. We already have PR for merge on this.